src/PHPStan/WPCliGetConfigDynamicReturnTypeExtension.php

```
@@ -0,0 +1,138 @@
+	}
+
+	/**
+	 * @return array<string, Type>
+	 */
+	private function getConfigMap(): array {
+		$stringType       = new StringType();
+		$stringOrNull     = TypeCombinator::addNull( $stringType );
+		$stringList       = new ArrayType( new IntegerType(), $stringType );
+		$boolType         = new BooleanType();
+		$trueOrStringList = TypeCombinator::union( new ConstantBooleanType( true ), $stringList );
+		$stringOrTrue     = TypeCombinator::union( $stringType, new ConstantBooleanType( true ) );
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$stringOrTrue'. (undefined)

(UnusedLocalVariable)